examples/vl3_basic/vl3-nse/cmd/vl3_connect.go

```diff
@@ -55,6 +55,7 @@ type vL3ConnectComposite struct {
 	//endpoint.BaseCompositeEndpoint
 	myEndpointName    string
 	nsConfig          *common.NSConfiguration
+	defaultCDPrefix   string
```

Comment:
IMO, the name for this var and the param to the constructor should be `fullIpCidr` or `defaultRouteIpCidr`

Reply:
done